FFmpeg.LGPL.x86.nuspec

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0"?>
+<package >
+  <metadata>
+    <id>FFmpeg.LGPL.x86</id>
+    <version>4.1.3-Final</version>
```

Comment:
Is `-Final` on the version a normal thing? It seems superfluous.

Reply:
It is, but I need a way to distinguish it from the one I already published during testing. Nugget doesn't let you delete or over write packages. Another solution would be to change the name of our package.